Makefile

```diff
@@ -27,50 +24,22 @@ single-test: build ## run a single test. usage: "make single-test test=[test nam
 build-release: ## Build release version
 	dotnet build -c=Release -f=netcoreapp2.0
 
-.PHONY: generate-env-file
-generate-env-file: ## Generate the environment file for running the tests inside a Docker container
-	scripts/generate_docker_env.sh
+.PHONY: build-package
+build-package: build-release ## Build and package NuGet
+	dotnet pack -c=Release ./src/GovukNotify/GovukNotify.csproj /p:TargetFrameworks=netcoreapp2.0 -o=publish
+
+.PHONY: bootstrap-with-docker
+bootstrap-with-docker:  # Prepare the Docker builder image
```

Comment:
This is missing the extra `#` needed to make the task show when you run `make help`

Reply:
Good spot! I've amended the second commit to fix that 👍.